stackslib/src/chainstate/nakamoto/tests/node.rs

```diff
@@ -373,93 +370,90 @@ impl TestStacksNode {
         burn_amount: u64,
         tenure_change_cause: TenureChangeCause,
     ) -> (LeaderBlockCommitOp, TenureChangePayload) {
-        let (
-            last_tenure_id,
-            previous_tenure_end,
-            previous_tenure_blocks,
-            parent_block_snapshot_opt,
-        ) = if let Some(parent_blocks) = parent_nakamoto_tenure {
-            // parent is an epoch 3 nakamoto block
-            let first_parent = parent_blocks.first().unwrap();
-            let last_parent = parent_blocks.last().unwrap();
-            let parent_tenure_id = StacksBlockId::new(
-                &first_parent.header.consensus_hash,
-                &first_parent.header.block_hash(),
-            );
-            let parent_sortition = SortitionDB::get_block_snapshot_consensus(
-                &sortdb.conn(),
-                &first_parent.header.consensus_hash,
-            )
-            .unwrap()
-            .unwrap();
+        let (last_tenure_id, previous_tenure_end, previous_tenure_blocks, parent_block_snapshot) =
+            if let Some(parent_blocks) = parent_nakamoto_tenure {
+                // parent is an epoch 3 nakamoto block
+                let first_parent = parent_blocks.first().unwrap();
+                let last_parent = parent_blocks.last().unwrap();
+                let parent_tenure_id = StacksBlockId::new(
+                    &first_parent.header.consensus_hash,
+                    &first_parent.header.block_hash(),
+                );
+                let parent_sortition = SortitionDB::get_block_snapshot_consensus(
+                    &sortdb.conn(),
+                    &first_parent.header.consensus_hash,
+                )
+                .unwrap()
+                .unwrap();
 
-            test_debug!(
-                "Work in {} {} for Nakamoto parent: {},{}",
-                burn_block.block_height,
-                burn_block.parent_snapshot.burn_header_hash,
-                parent_sortition.total_burn,
-                last_parent.header.chain_length + 1,
-            );
+                test_debug!(
```

Comment:
The diff seems to be implying the indentation is weird here, was this properly `cargo fmt`'d?